source/features/more-dropdown.tsx

```diff
@@ -17,7 +17,7 @@ const repoUrl = getRepoURL();
 
 function createDropdown(): void {
 	// Markup copied from native GHE dropdown
-	appendBefore('.reponav', '[data-selected-links^="repo_settings"]',
+	appendBefore('.reponav > ul', '[data-selected-links^="repo_settings"]',
```

Comment:
GHE (2.19) still doesn't have the `ul` types and this will break the feature. I know we don't care most of the time about GHE, but these two features are used a lot and will break a lot of people and this is why we currently don't have the `ul` check in there. Can we check if the `ul` exists first or use `isEnterprise()` instead?

I think we're now at 2+ years of GitHub having swapped to the `ul` type on GH, but not ported it over to GHE yet. 😭